var/spack/repos/builtin/packages/adol-c/package.py

```diff
@@ -30,6 +30,8 @@ class AdolC(AutotoolsPackage):
             description='Enable advanced branching to reduce retaping')
     variant('atrig_erf', default=True,
             description='Enable arc-trig and error functions')
+    variant('stdczero', default=True,
+            description='Enable default initialization for the adouble datatype')
```

Comment:
Is this a typo:
```suggestion
            description='Enable default initialization for the double datatype')
```
or there is an `adouble` datatype?

Reply:
The type is actually `adouble`, which is the AD version of a plain `double`.